app/serializers/export/post_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module Export
+  class PostSerializer < ActiveModel::Serializer
+    attributes :text,
+               :author_id,
+               :public,
+               :diaspora_handle,
+               :type,
+               :image_url,
+               :image_height,
+               :image_width,
+               :likes_count,
+               :comments_count,
+               :reshares_count,
+               :favorite
```

Comment:
Interesting, first time seen this and doesn't to me look like it's used. There is a `toggleFavorite` method in the JS but it's not called :P

Some legacy idea that never got implemented, @jhass ? I'd personally remove it if we don't use the attribute.